lib/rentlinx/validators/phone_validator.rb

```diff
@@ -0,0 +1,14 @@
+module Rentlinx
+  class PhoneValidator < BaseValidator
+    def validate
```

Comment:
`processed_value` is used outside of the validator. The result of `validate` is cached in `@error` and is only run on initialize, so it can be made private.